src/test/java/htsjdk/tribble/AbstractFeatureReaderTest.java

```diff
@@ -27,6 +34,19 @@
     final static String HTTP_INDEXED_VCF_PATH = TestUtil.BASE_URL_FOR_HTTP_TESTS + "ex2.vcf";
     final static String LOCAL_MIRROR_HTTP_INDEXED_VCF_PATH = VariantBaseTest.variantTestDataRoot + "ex2.vcf";
 
+    //the "mangled" versions of the files have an extra byte added to the front of the file that makes them invalid
+    private static final String TEST_PATH = "src/test/resources/htsjdk/tribble/AbstractFeatureReaderTest/";
+    private static final String MANGLED_VCF = TEST_PATH + "mangledBaseVariants.vcf";
+    private static final String MANGLED_VCF_INDEX = TEST_PATH + "mangledBaseVariants.vcf.idx";
+    private static final String VCF = TEST_PATH + "baseVariants.vcf";
+    private static final String VCF_INDEX = TEST_PATH + "baseVariants.vcf.idx";
+    private static final String VCF_TABIX = TEST_PATH + "baseVariants.vcf.gz";
+    private static final String VCF_TABIX_INDEX = TEST_PATH + "baseVariants.vcf.gz.tbi";
+    private static final String MANGLED_VCF_TABIX = TEST_PATH + "baseVariants.mangled.vcf.gz";
```

Comment:
Can you name block-gzipped test inputs to make it clear in the name that they are block-gzipped rather than regular-gzipped?